packages/components/src/components/CodeEditor/index.tsx

```diff
@@ -17,10 +12,8 @@ export type CodeEditorProps = {
   onSubmit?: () => void;
   // can be used as a hotkey (Cmd+Option+V, see `useViewNodeExtension`) or as an action for gutter markers
   onFocusByPath?: (path: SqValuePath) => void;
-  width?: number;
   height?: number | string;
   lineWrapping?: boolean;
-  errors?: SqError[];
```

Comment:
Note I've removed `width` and `errors`, we weren't using `width`, and `errors` can be derived from `simulation`.